### PR TITLE
Add another inline via jquery

### PR DESCRIPTION
**Pull request by preston-wagner**

### CODE REVIEW

File: viewsets/templates/base/create.html [hunk 1]
```diff
@@ -15,15 +15,40 @@
...
+                    console.log($forms_list[0]);
+                    $forms_list.append(
+                        $empty_form.clone().removeClass('empty-form').addClass('form').html(
+                            $empty_form.html().replace(/__prefix__/g, form_idx)
+                        )
+                    );
+                    $total_forms.val(parseInt(form_idx) + 1);
+                });
+            });
+        </script>
```
Comment (freakypie) [inline viewsets/templates/base/create.html]: isn't there a "extra_scripts" block? that's probably where this should go.

Comment (preston-wagner) [reply viewsets/templates/base/create.html]: ok, it's there now.

File: viewsets/templates/base/create.html [hunk 2]
```diff
@@ -105,4 +118,26 @@
...
+    <script>
+        $(function(){
+            $('.form-inline > .add-more').click(function(event){
+                event.preventDefault();
+                var $inline_form = $(this).parent();
+                var $total_forms = $inline_form.find('[name*="TOTAL_FORMS"]');
+                var form_idx = $total_forms.val();
+                var $empty_form = $inline_form.find('.empty-form');
+                var $forms_list = $inline_form.find('.forms');
+                console.log($forms_list[0]);
```
Comment (freakypie) [inline viewsets/templates/base/create.html]: we can't publish this with `console.log` :(

**Comment by freakypie** [comment]
If we are changing "extrascripts" then it needs to be changed for all the templates. I recommend just reverting that part. We can merge it in and change the name later

**Pull request merged**